From 28d4bdd8c1c4291f9d104d3521ce59876718a23b Mon Sep 17 00:00:00 2001
From: Zev Weiss <zev@bewilderbeest.net>
Date: Sat, 26 Sep 2020 16:27:34 -0500
Subject: [PATCH 8/8] peci-cputemp: label CPU cores from zero instead of one

Zero-based numbering is more consistent with all other cpu/core
numbering I'm aware of (including the PECI spec).

OpenBMC-Staging-Count: 1
Signed-off-by: Zev Weiss <zev@bewilderbeest.net>
Reviewed-by: Jae Hyun Yoo <jae.hyun.yoo@linux.intel.com>
Link: https://lore.kernel.org/r/20200926212734.23836-3-zev@bewilderbeest.net
Signed-off-by: Joel Stanley <joel@jms.id.au>
---
 drivers/hwmon/peci-cputemp.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/hwmon/peci-cputemp.c b/drivers/hwmon/peci-cputemp.c
index b9fe912..78e442f 100644
--- a/drivers/hwmon/peci-cputemp.c
+++ b/drivers/hwmon/peci-cputemp.c
@@ -363,7 +363,7 @@ static int create_core_temp_label(struct peci_cputemp *priv, int idx)
 	if (!priv->coretemp_label[idx])
 		return -ENOMEM;
 
-	sprintf(priv->coretemp_label[idx], "Core %d", idx + 1);
+	sprintf(priv->coretemp_label[idx], "Core %d", idx);
 
 	return 0;
 }
-- 
2.7.4

